mailbox/event/event-cassandra/src/test/java/org/apache/james/mailbox/events/CassandraEventDeadLettersTest.java

```diff
@@ -0,0 +1,53 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.events;
+
+import org.apache.james.event.json.EventSerializer;
+import org.apache.james.mailbox.events.EventDeadLettersContract.FailedEventContract;
+import org.apache.james.mailbox.events.EventDeadLettersContract.FailedEventsContract;
+import org.apache.james.mailbox.events.EventDeadLettersContract.GroupsWithFailedEventsContract;
+import org.apache.james.mailbox.events.EventDeadLettersContract.RemoveContract;
+import org.apache.james.mailbox.events.EventDeadLettersContract.StoreContract;
+import org.apache.james.mailbox.model.TestId;
+import org.apache.james.mailbox.model.TestMessageId;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.datastax.driver.core.Session;
+
+class CassandraEventDeadLettersTest implements StoreContract, RemoveContract, FailedEventContract, FailedEventsContract,
+    GroupsWithFailedEventsContract {
+
+    @RegisterExtension
+    static CassandraEventDeadLettersExtension deadLettersExtension = new CassandraEventDeadLettersExtension();
```

Comment:
I prefer having simply the CassandraClusterExtension being registered here, and dropping the code in `CassandraEventDeadLettersExtension`

Reply:
Yes, I will. srr, I did not test carefully.